app/controllers/main_controller.rb

```diff
@@ -0,0 +1,4 @@
+class MainController < ApplicationController
```

Comment:
Just double checking, whats the main controller? Is it similar to the welcome one in our previous repos.